src/transformers/tokenization_utils_base.py

```diff
@@ -2082,6 +2082,14 @@ def save_pretrained(
         )
 
         tokenizer_config = copy.deepcopy(self.init_kwargs)
+
+        # TODO: Ensure the modified attributes (those are also in the __init__ kwargs) will give identical tokenizers
+        # target_keys = self.init_kwargs.keys()
+        target_keys = ["model_max_length"]
```

Comment:
I don't want to touch other attributes at this moment.
(although CircleCI only has 3 or 4  failures from T5 if I use `self.init_kwargs.keys()`)